src/Pact/Native.hs

```diff
@@ -138,12 +139,14 @@ natives =
   , guardDefs
   , zkDefs
   , hashDefs
-  , hyperlaneDefs
   ]
 
 -- | Production native modules as a dispatch map.
 nativeDefs :: HM.HashMap Text Ref
-nativeDefs = mconcat $ map moduleToMap natives
+nativeDefs = mconcat $ map moduleToMap (hyperlaneDefs:natives)
```

Comment:
I see this is a leftover from the old code, but a tiny-tiny nitpick: one could `foldMap` instead of `mconcat $ map`.